RELEASENOTES-1.4.docu

```diff
@@ -474,4 +474,12 @@
       invalid generated C when both objects are targets of <tt>each
       .. in ..</tt> expressions
       <bug number="SIMICS-21044"/>.</add-note></build-id>
+  <build-id value="next"><add-note>It's now allowed to cast an expression
+      of a struct type to that same struct type.</add-note></build-id>
+  <build-id value="next"><add-note>Fixed an error where const method parameters
+      would cause an internal error if that method is used as the callback of
+      an `after` statement.</add-note></build-id>
+  <build-id value="next"><add-note>Fixed an issue where having a const-qualified
+      layout type could lead to internal compiler errors in certain
+      scenarios.</add-note></build-id>
```

Comment:
Do these largely-independent microissues deserve JIRA issues? Or maybe a single JIRA issue for them should be created collectively, despite the fact they're independent?

Reply:
I'd say zero or one jira makes sense. It's kind of nice to have a jira for historic reference, but the use is probably limited in practice (unclear how much effort we'll put into publicizing history if we move to a public issue tracker).